### PR TITLE
release: updating stable release instructions and owner

### PR DESCRIPTION
**Pull request by alyssawilk**
Risk Level: n/a
Testing: n/a
Docs Changes: yes
Release Notes: no

### CODE REVIEW

**Comment by alyssawilk** [comment]
@lizan tells me @cpakulski is up for being stable maintainer this quarter - updating docs accordingly.  Thanks so much Christoph!

File: RELEASES.md
Comment (lizan) [inline RELEASES.md:55]: Also, Rei Shimizu (@Shikugawa) agreed to be signed up in the next release. We can leave this updated at release 1.17 or just update now.

**Review by lizan — APPROVED** [review]
Thank you @cpakulski and @Shikugawa!

File: RELEASES.md
Comment (alyssawilk) [reply RELEASES.md]: Oh awesome, thanks @Shikugawa 
Also a huge thanks to @lambdai and @PiotrSikora for their work thusfar!  :-)

File: RELEASES.md
Comment (PiotrSikora) [inline RELEASES.md:56]: Those links don't point anywhere right now (see the last line in this file). I think it might be better to inline links here, otherwise we keep missing the link targets.

**Review by PiotrSikora — CHANGES_REQUESTED** [review]

File: RELEASES.md
Comment (alyssawilk) [reply RELEASES.md:56]: yeah, good call, I'd missed those entirely

Comment (PiotrSikora) [reply RELEASES.md:56]: Nit: `s/shikugawa/Shikugawa/`

**Review by PiotrSikora — APPROVED** [review]
LGTM, once Rei's GitHub is fixed.

**Review by PiotrSikora — APPROVED** [review]
Thanks!

**Review by lizan — APPROVED** [review]

**Pull request merged**
